app/assets/javascripts/discourse/app/components/glimmer-header/topic/info.gjs

```diff
@@ -0,0 +1,187 @@
+import Component from "@glimmer/component";
+import { fn, hash } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import categoryLink from "discourse/helpers/category-link";
+import concatClass from "discourse/helpers/concat-class";
+import renderTags from "discourse/lib/render-tags";
+import DiscourseURL from "discourse/lib/url";
+import icon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import and from "truth-helpers/helpers/and";
+import gt from "truth-helpers/helpers/gt";
+import not from "truth-helpers/helpers/not";
+import or from "truth-helpers/helpers/or";
+import PluginOutlet from "../../plugin-outlet";
+import FeaturedLink from "./featured-link";
+import Participant from "./participant";
+import Status from "./status";
+
+export default class Info extends Component {
+  @service currentUser;
+  @service site;
+  @service siteSettings;
+
+  get showPM() {
+    return !this.args.topic.is_warning && this.args.topic.isPrivateMessage;
+  }
+
+  get totalParticipants() {
+    return (
+      (this.args.topic.details.allowed_users?.length || 0) +
+      (this.args.topic.allowed_groups?.length || 0)
+    );
+  }
+
+  get maxExtraItems() {
+    return this.args.topic.tags?.length > 0 ? 5 : 10;
+  }
+
+  get additionalFancyTitleClasses() {
+    return _additionalFancyTitleClasses.join(" ");
```

Comment:
`_additionalFancyTitleClasses` is not defined in this file

Reply:
Removed in https://github.com/discourse/discourse/pull/25728